firefox-ios/Client/AccessoryViewProvider.swift

```diff
@@ -163,54 +141,42 @@ class AccessoryViewProvider: UIView, Themeable {
     }
 
     private func setupLayout() {
-        translatesAutoresizingMaskIntoConstraints = false
         setupSpacer(leadingFixedSpacer, width: UX.fixedLeadingSpacerWidth)
         setupSpacer(trailingFixedSpacer, width: UX.fixedTrailingSpacerWidth)
 
-        if showCreditCard {
-            let cardStackViewForBarButton = UIBarButtonItem(customView: cardButtonStackView)
-            cardStackViewForBarButton.accessibilityTraits = .button
-            cardStackViewForBarButton.accessibilityLabel = .CreditCard.Settings.UseSavedCardFromKeyboard
```

Comment:
This accessibility label got removed, but I don't see it elsewhere. Was the deletion intentional? If so, curious why